contrib/internal/httputil/trace.go

```diff
@@ -17,30 +17,44 @@ import (
 	"gopkg.in/DataDog/dd-trace-go.v1/ddtrace/tracer"
 )
 
+// TraceConfig defines the configuration for request tracing.
+type TraceConfig struct {
+	Writer         http.ResponseWriter       // response writer
+	Request        *http.Request             // request that is traced
+	Service        string                    // service name
+	Resource       string                    // resource name
+	QueryParamTags bool                      // specifies that request query parameters should be appended to http.url tag
```

Comment:
```suggestion
	QueryParams bool                      // specifies that request query parameters should be appended to http.url tag
```
There are no tags anymore, we are now just appending to `http.url`